internal/rbd/driver/driver.go

```diff
@@ -125,13 +126,17 @@ func (r *Driver) Run(conf *util.Config) {
 			})
 	}
 
-	if conf.EnableReadAffinity {
-		crushLocationMap, err = util.GetCrushLocationMap(conf.CrushLocationLabels, conf.NodeID)
+	if conf.ContainerOrchestrator == "kubernetes" {
```

Comment:
if `ContainerOrchestrator` is only used here, consider replacing it with `internal/util/k8s.RunsOnKubernetes()` or similar